src/uct/ib/base/ib_md.h

```diff
@@ -496,6 +496,13 @@ ucs_status_t uct_ib_reg_mr(struct ibv_pd *pd, void *addr, size_t length,
 ucs_status_t uct_ib_dereg_mr(struct ibv_mr *mr);
 ucs_status_t uct_ib_dereg_mrs(struct ibv_mr **mrs, size_t mr_num);
 
+
+/**
+ * Check if IB md device has ECE capability
```

Comment:
Set the field md->ece_support to 1 if the device supports ECE capability, otherwise set it to 0.
Returns an error if failed to check for ECE support.

Reply:
Done